src/domain/project/project.interface.ts

```diff
@@ -1,4 +1,5 @@
 import { IAgreement } from '../agreement/agreement.interface';
+import { IAspect } from '../aspect/aspect.interface';
 import { ITagset } from '../tagset/tagset.interface';
 
 export interface IProject {
```

Comment:
This might be the place where **textId** is missing. Will we be able to query groups and users for a project?